### PR TITLE
Kiwix banner is now under scrollable region and All

### LINKED ISSUE #1150: Feature Ehancement: Convert Material Design png icons into vector assets
Should i Work on this
Most image formats (png, jpeg, bmp, gif, webp etc) are raster which means they describe the image as a fixed grid of pixels.
Using Vector assets will have benefits like
Sharp/Vector images Resize gracefully
Small
Dynamic

### LINKED ISSUE #1355: Add main Kiwix banner inside the scrollable region
**Is your feature request related to a problem? Please describe.**
Making the banner scrollable would add extra space for displaying the zim files.

**Describe the solution you'd like**

Just add the main banner inside the scroll view (along with the recycler view).

### PR DESCRIPTION
**Pull request by rajput2122**
Fixes #1355 
Fixes #1150

Changes:i have changed layout of main activity in which i placed main kiwix banner into scrollview along with recyclerview
Screenshot_20190902-033457_Kiwix
Screenshot_20190902-033503_Kiwix

Screenshots/GIF for the change: [If possible, please add relevant screenshots/GIF]

### CODE REVIEW

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -1,248 +1,30 @@
 <component name="ProjectCodeStyleConfiguration">
   <code_scheme name="Project" version="173">
-    <option name="USE_SAME_INDENTS" value="true" />
-    <option name="IGNORE_SAME_INDENTS_FOR_LANGUAGES" value="true" />
-    <option name="OTHER_INDENT_OPTIONS">
-      <value>
-        <option name="INDENT_SIZE" value="2" />
-        <option name="CONTINUATION_INDENT_SIZE" value="4" />
-        <option name="TAB_SIZE" value="2" />
```
Comment (abdulwd) [inline .idea/codeStyles/Project.xml]: Why have you deleted this? Revert these changes.

Comment (rajput2122) [reply .idea/codeStyles/Project.xml]: I don't delete this because i didn't open it while fixing

Comment (rajput2122) [reply .idea/codeStyles/Project.xml]: how do i revert kindly guide me?

**Comment by rajput2122** [comment]
@abdulwd sir please tell me what i have to do in this pull request

File: .idea/codeStyles/Project.xml (hunk 1 above)
Comment (rajput2122) [reply .idea/codeStyles/Project.xml]: sir i have update my commit actually i committed the whole directory in which project.xml got modified now i committed adding files individually but travis build error is coming.Can you please help me to resolve it?

**Comment by macgills** [comment]
Due to security reasons we can not allow builds from forks to run to completion. Once this has gotten approval I will open a branch on the main repo, merge this PR to it and then create a new PR so the build can run

**Comment by rajput2122** [comment]
@abdulwd i had made the requested change can you please review it?
